core/domain/exp_jobs_one_off.py

```diff
@@ -507,7 +509,41 @@ def reduce(key, values):
 
 class ExplorationMigrationValidationJob(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
For consistency should you add "TextAngular" to the title of this class?

Or are you planning to include the CKEditor migration-validation together in the same place?

Reply:
Yes, I am planning to do that in a separate PR.